src/gallery/fullscreen.js

```diff
@@ -28,8 +29,41 @@ const renderFullScreenGallery = ( lang, dir ) => {
 					</div>
 					<div class="wp-gallery-fullscreen-button next"></div>
 				</div>
-				<div class="wp-gallery-fullscreen-bottom">
-					<div class="wp-gallery-fullscreen-caption"></div>
+				<div class="wp-gallery-fullscreen-bottom"></div>
+			</div>
+		`.trim()
+	},
+
+	renderImageInfo = ( mediaInfo, container, image, lang ) => {
+		container.innerText = ''
+		const getImageDescription = () => {
+				if ( image.caption ) {
+					return image.caption
+				} else if ( mediaInfo.description ) {
+					return mediaInfo.description
+				} else {
+					return ''
+				}
+			},
+
+			license = mediaInfo.license,
+			author = mediaInfo.author ? mediaInfo.author : msg( lang, 'gallery-unknown-author' ),
+			link = mediaInfo.filePage,
+			description = getImageDescription()
+
+		return `
+			<div class="wp-gallery-fullscreen-caption">${description}</div>
+			<div class="wp-gallery-fullscreen-attribution">
+				<div class="wp-gallery-fullscreen-attribution-info">
+					${ license && license.indexOf( 'CC' ) !== -1 ? '<div class="wp-gallery-fullscreen-attribution-info-cc"></div>' : ''}
```

Comment:
There must be a cleaner way to treat those 5 cases

Reply:
Yea agreed, how about now with `getLicenseInfo`?